CMakeLists.txt

```diff
@@ -61,8 +61,6 @@ catkin_package(
   LIBRARIES loam 
 )
 
-add_definitions( -march=native )
-
```

Comment:
how does this help?

Reply:
This stops a runtime error. See https://github.com/RMFHOPE/RAMP#troubleshooting